### PR TITLE
Organized imports / Refactoring

### PR DESCRIPTION
**Pull request by svenkubiak**
Nothing fancy. I organized the imports on all projects, refactored some deprecated methods and took the liberty of adding lukas eichler and me to the team.

### CODE REVIEW

File: ninja-core/src/main/java/ninja/WrappedContext.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public WrappedContext(Context wrapped) {
 
     @Override
     public String getRequestUri() {
-        return wrapped.getRequestUri();
```
Comment (raphaelbauer) [inline ninja-core/src/main/java/ninja/WrappedContext.java:47]: please leave it that way - it's intentionally referencing the deprecated version.

File: ninja-core/src/main/java/ninja/WrappedContext.java [hunk 2]
```diff
@@ -64,12 +64,12 @@ public String getRemoteAddr() {
 
     @Override
     public FlashScope getFlashCookie() {
-        return wrapped.getFlashCookie();
```
Comment (raphaelbauer) [inline ninja-core/src/main/java/ninja/WrappedContext.java:67]: please leave it that way - it's intentionally referencing the deprecated version.

File: ninja-core/src/main/java/ninja/WrappedContext.java [hunk 3]
```diff
@@ -64,12 +64,12 @@ public String getRemoteAddr() {
...
+        return wrapped.getFlashScope();
     }
 
     @Override
     public Session getSessionCookie() {
-        return wrapped.getSessionCookie();
```
Comment (raphaelbauer) [inline ninja-core/src/main/java/ninja/WrappedContext.java:72]: please leave it that way - it's intentionally referencing the deprecated version.

File: ninja-core/src/test/java/ninja/utils/SimpleClass.java [hunk 4]
```diff
@@ -16,6 +16,8 @@
 
 package ninja.utils;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```
Comment (raphaelbauer) [inline ninja-core/src/test/java/ninja/utils/SimpleClass.java]: why is that necessary?

File: ninja-servlet-integration-test/src/main/java/etc/GreetingServiceImpl.java [hunk 5]
```diff
@@ -22,7 +22,8 @@
 
     private final long initTime = new Date().getTime();
 
-	public String hello() {
+	@Override
```
Comment (raphaelbauer) [inline ninja-servlet-integration-test/src/main/java/etc/GreetingServiceImpl.java]: indentation wrong

File: ninja-servlet-integration-test/src/main/java/etc/GreetingServiceTestImpl.java [hunk 6]
```diff
@@ -22,7 +22,8 @@
 
     private final long initTime = new Date().getTime();
 
-	public String hello() {
+	@Override
```
Comment (raphaelbauer) [inline ninja-servlet-integration-test/src/main/java/etc/GreetingServiceTestImpl.java]: indentation wrong

**Comment by raphaelbauer** [comment]
Very nice clean-up! I added some remarks. Once that's done we'll merge it asap :)

**Comment by raphaelbauer** [comment]
One other tiny comment: Please make sure that the PRs only contain one topic. If multiple things are fixed in one PR it becomes really hard to review (names in team.md, deprecated methods refactored, documentation somewhere else, reorganizing of imports ...) :) Thanks!

File: ninja-core/src/test/java/ninja/utils/SimpleClass.java (hunk 4 above)
Comment (svenkubiak) [inline ninja-core/src/test/java/ninja/utils/SimpleClass.java]: Looks like the link to @objectmapper in the comment has triggered the import by the automatic clean up. Removed it.

File: ninja-servlet-integration-test/src/main/java/etc/GreetingServiceImpl.java (hunk 5 above)
Comment (svenkubiak) [inline ninja-servlet-integration-test/src/main/java/etc/GreetingServiceImpl.java]: looked okay on my eclipse. can you check the updated version?

File: ninja-servlet-integration-test/src/main/java/etc/GreetingServiceTestImpl.java (hunk 6 above)
Comment (svenkubiak) [inline ninja-servlet-integration-test/src/main/java/etc/GreetingServiceTestImpl.java]: looked okay on my eclipse. can you check the updated version?

**Comment by svenkubiak** [comment]
@raphaelbauer I have updated the PR. 

And yes, you are right. Should have made separate pull-requests for the changes. Sorry ;-)

**Comment by raphaelbauer** [comment]
Looks perfect :) Thanks!
